src/images/window-buttons-disabled.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 8 2" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We can likely do these circles as pseudo elements in css, right? Try a single `::before` element with multiple background shadows for the correct effect:

```css
.whatever::before {
  content: "";
  background-color: the-correct-yellow-color;
  background-shadow: 0 -5.6rem 0 the-correct-green, 0 5.6rem 0 the-correct-red;
}
```
Or something like that.

Reply:
Punting, if you don't mind